ios/AirshipFrameworkProxy/AirshipProxyEventEmitter.swift

```diff
@@ -1,7 +1,8 @@
 import Foundation
 @preconcurrency public import Combine
 
-public actor AirshipProxyEventEmitter {
+@MainActor
+public final class AirshipProxyEventEmitter {
```

Comment:
By moving this to the main actor it makes plugins that are not swift6 friendly easier to manage